### PR TITLE
Update docs.

### PR DESCRIPTION
**Pull request by ensonic**
Update the docs for the docker_rules.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -28,10 +28,10 @@ git_repository(
 
 load(
   "@io_bazel_rules_docker//docker:docker.bzl",
```
Comment (mattmoor) [inline README.md]: We should load this from `container:container.bzl` if we're going to use this name (which we should!).

Comment (ensonic) [reply README.md]: Well spotted. Thanks and fixed.

File: README.md [hunk 2]
```diff
@@ -203,7 +203,6 @@ the following:
 ```shell
 $ cat .bazelrc
 build --workspace_status_command=./print-workspace-status.sh
-run --workspace_status_command=./print-workspace-status.sh
```
Comment (mattmoor) [inline README.md]: Why did you remove this?

Comment (ensonic) [reply README.md]: See commit message:
Simplify .bazelrc example.  …
According to
https://docs.bazel.build/versions/master/command-line-reference.html#run
run inherits from build and therefore we can omit the entry for run.

**Comment by mattmoor** [comment]
Cool thanks.  Travis only runs properly from PRs coming from branches on the main repo (b/c of secrets), and I'm just going to skip Bazel CI because this is README only.
